action.yml

```diff
@@ -10,25 +10,25 @@ inputs:
     required: true
   GITHUB_TOKEN:
     description: "Github authentication token that allows comments to be created on PRs (optional)"
+    default: ${{ github.token }}
     required: false
   SHOPIFY_STORE_URL:
     description: "The shopify development store i.e. my-store.myshopify.com"
     required: true
   SHOPIFY_PASSWORD:
     description: "The Shopify store's private app password used with themekit"
     required: true
-  SHOPIFY_API_KEY:
-    description: "The Shopify store's private app API Key to allow theme creation and removal (if this is opted for)"
-    required: true
   SHOPIFY_THEME_ID:
     description: "If provided the theme will be deployed to this Shopify theme id. If not provided the theme will be deployed to a PR specific newly created theme."
     required: false
   SHOPIFY_THEME_DIRECTORY:
     description: "The directory containing the Shopify theme to deploy i.e. './dist'"
-    required: true
+    required: false
+    default: "."
   SHOPIFY_ALLOW_LIVE_THEME_DEPLOYMENT:
     description: "If deploying to a 'live' Shopify theme this must be set to 'true'"
     required: false
+    default: "false"
```

Comment:
Suspect required now needs to change to `false` as it has a default value

Reply:
@matthew-petrie 
Could you extend your thought?
I belive, default value was needed here because of null exception